### PR TITLE
Add IBM Busines Automation Workflow

### PR DESCRIPTION
**Pull request by hubje2020**
Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - Status: please select a value from the status table at the top
  - Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - Please mind the sorting
  - Please put vendor/product name in PR title (instead of "Update README.md")

### CODE REVIEW

File: software/README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 | Apache        | Hadoop | 3.3.1 | Vulnerable | Assumed vulnerable, log4j is used. |[source](https://gist.github.com/francescosimoneschi/6530036) |
 | Apache        | James | 3.6.0 | Vulnerable |  |[source](https://twitter.com/dlitchfield/status/1469809966785564675) |
 | Apache        | Jena | < 4.3.1 | Fix | Fixed in 4.3.1 |[source](https://blogs.apache.org/security/entry/cve-2021-44228)|
+| Apache        | JMeter | Any | Vulnerable| Manual Bypass |[source](https://github.com/apache/jmeter/pull/680#issuecomment-993204248)|
```
Comment (maertsen) [inline software/README.md:89]: I think what you describe is a Workaround. Could you update the status field, empty the comment and rename the url from source to workaround?

File: software/README.md [hunk 2]
```diff
@@ -1026,6 +1027,7 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 | Supplier        | Product         | Version (see Status) | Status          | Notes           | Links |
 |:----------------|:----------------|:--------------------:|:---------------:|:----------------|-----------:|
 | IBM | All products | | Investigation | | [source](https://www.ibm.com/blogs/psirt/an-update-on-the-apache-log4j-cve-2021-44228-vulnerability/)|
+| IBM | Business Automation Workflow | 18.0.0+ | Vulnerable | Fixes | [source](https://www.ibm.com/support/pages/node/6525834)|
```
Comment (maertsen) [inline software/README.md:1030]: Please choose between Vulnerable or Fix; list the vulnerable (or fixed) versions in the version field. Thanks!

**Review by maertsen — CHANGES_REQUESTED** [review]
thanks for your contribution @hubje2020. Can you have a look at my comments?

File: software/README.md (hunk 2 above)
Comment (hubje2020) [reply software/README.md]: Fixed

**Pull request merged**